frontend/src/tests/pages/AdminDisplayTablePage.test.js

```diff
@@ -0,0 +1,184 @@
+        );
+
+        await waitFor(() => { expect(getByTestId(`${testId}-cell-row-0-col-id`)).toHaveTextContent("5"); });
+        expect(getByTestId(`${testId}-cell-row-1-col-id`)).toHaveTextContent("4");
+        expect(getByTestId(`${testId}-cell-row-2-col-id`)).toHaveTextContent("1");
+
+    });
+
+
+    // test("renders empty table when backend unavailable", async () => {
+
+    //     const queryClient = new QueryClient();
+    //     axiosMock.onGet("/api/commons/all").timeout();
+
+    //     const restoreConsole = mockConsole();
+
+    //     const { queryByTestId } = render(
+    //         <QueryClientProvider client={queryClient}>
+    //             <MemoryRouter>
+    //                 <AdminDisplayTablePage />
+    //             </MemoryRouter>
+    //         </QueryClientProvider>
+    //     );
+
+    //     await waitFor(() => { expect(axiosMock.history.get.length).toBeGreaterThanOrEqual(1); });
+
+    //     const errorMessage = console.error.mock.calls[0][0];
+    //     expect(errorMessage).toMatch("Error communicating with backend via GET on /api/commons/all");
+    //     restoreConsole();
+
+    //     expect(queryByTestId(`${testId}-cell-row-0-col-id`)).not.toBeInTheDocument();
+    // });
+
+    test("test what happens when you click delete", async () => {
+
+        const queryClient = new QueryClient();
+        axiosMock.onGet("/api/commons/all").reply(200, commonsFixtures.threeCommons);
+        axiosMock.onDelete("/api/commons/delete").reply(200, "record 5 was deleted");
+
+
+        const { getByTestId } = render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <AdminDisplayTablePage />
+                </MemoryRouter>
+            </QueryClientProvider>
+        );
+
+        await waitFor(() => { expect(getByTestId(`${testId}-cell-row-0-col-id`)).toBeInTheDocument(); });
+
+       expect(getByTestId(`${testId}-cell-row-0-col-id`)).toHaveTextContent("5"); 
+
+
+        const deleteButton = getByTestId(`${testId}-cell-row-0-col-Delete-button`);
+        expect(deleteButton).toBeInTheDocument();
+       
+        fireEvent.click(deleteButton);
+
+        await waitFor(() => { expect(mockToast).toBeCalledWith("record 5 was deleted") });
+
+    });
+
+    // test("When you fill in form and click submit, the right things happens", async () => {
+
+    //     axiosMock.onPost("/api/commons/new").reply(200, {
+    //         "id": 5,
+    //         "name": "Seths Common",
+    //         "day": 5,
+    //         "endDate": "6/11/2021",
+    //         "totalPlayers": 50,
+    //         "cowPrice": 15,
+    //     });
+
+    //     const { getByText, getByLabelText, getByTestId } = render(
+    //         <QueryClientProvider client={queryClient}>
+    //             <MemoryRouter>
+    //                 <AdminCreateCommonsPage />
+    //             </MemoryRouter>
+    //         </QueryClientProvider>
+    //     );

```

Comment:
What is the purpose of this commented out code?

Reply:
deleted commented out lines